LeetCodeAssistant/LeetCodeProblemRepository.swift

```diff
@@ -314,3 +367,9 @@ fileprivate func sanitizeMemoryUsage(_ memoryUsage: String) -> String {
     
     return memoryUsage
 }
+
+fileprivate extension String {
+    func removeHtmlTag() -> String {
```

Comment:
This extension method is only used in one place. I don't think it needs to be extension for String. How about just defining as utility function?